index.ts

```diff
@@ -16,6 +16,7 @@ const ITokenHandler = require('./build/contracts/ITokenHandler.json');
 const IVersionRegistry = require('./build/contracts/IVersionRegistry.json');
 const IWalletCustomLogic = require('./build/contracts/IWalletCustomLogic.json');
 const IWalletFactory = require('./build/contracts/IWalletFactory.json');
+const ERC20 = require('./build/contracts/ERC20.json');
```

Comment:
@franciscotobar Just a question: does this artifact refer to [this one](https://github.com/rsksmart/rif-relay-contracts/blob/5c44f2bb6a49fe69686214110132088b402c53e0/contracts/test/tokens/Tether.sol#L81)?

Reply:
No it does not, it refers to this:

[ERC20](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.2.0/contracts/token/ERC20/ERC20.sol)